### PR TITLE
add MVP basic program

### PR DESCRIPTION
**Pull request by AntoniLuongPham**
Dear @laurensvalk, @dlech,

I am having school holiday so I am programming my robots again.

Please find my mvp basic program for your review.

I will program advanced mvp robots later when the remote control is available.
Now I will program the other robots that do not need the  remote control first, like Charlie, Blast.

Thank you very much! :)

Antoni

### CODE REVIEW

File: sets/inventor/mvp/mvp_basic.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    def calibrate(self):
+        self.steer_motor.run_target(speed=1000, target_angle=0)
+
+
+# Initialize mvp and straighten its steering.
+mvp = MVP()
+mvp.calibrate()
+
+# Make mvp drive in a cirle.
```
Comment (dlech) [inline sets/inventor/mvp/mvp_basic.py:25]: It looks like we should fix the spelling of "cirle" here.

File: sets/inventor/mvp/mvp_basic.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+mvp.steer_motor.run_angle(speed=350, rotation_angle=50)
+mvp.drive_motor.run_angle(speed=800, rotation_angle=-16 * 360)
```
Comment (dlech) [inline sets/inventor/mvp/mvp_basic.py:27]: Is this driving backwards since it has a negative angle?

**Comment by AntoniLuongPham** [comment]
Dear @dlech,

I have fixed the typo mistake and change the positive direction of the drive motor to make the program nicer.

Thank you very much for your review!

Antoni

**Comment by laurensvalk** [comment]
Thanks for this pull request and the others as well!

In the mean time, we have been working to update our website to make all these projects a bit more more accessible. Our plan is to publish most or all of the `pybricks-projects` over at http://pybricks.com/projects, to make it easier for people to find them.

For contributors, all it takes is adding a [few extra details to `README.md`](https://raw.githubusercontent.com/pybricks/pybricks-projects/master/sets/boost/auto-builder/README.md).

We've done one project as an example: https://github.com/pybricks/pybricks-projects/tree/master/sets/boost/auto-builder/
Once approved, they are published like this: https://pybricks.com/projects/sets/boost/auto-builder/

We are just getting started with this new format, so we're happy to take feedback. Would you be interested in updating one of your PRs for now? Curious to hear what you think.

**Comment by TheVinhLuong102** [comment]
Hi @laurensvalk,

Apologies for our delayed reply! -- @AntoniLuongPham and @ArianaLuongPham are back to in-person schooling and the schedule has been hectic. :)

Also, thank you for the recent merges!

I've taken a look at the markdown headers you proposed and those seem very doable, so I'll work with the kids to add such headers for the project publications. We may need to re-film several videos, as the quality of those done in the past has been hit-and-miss (I just focused on getting them to talk, and didn't prioritize filming quality). :D

Vinh

**Pull request merged**